README.md

```diff
@@ -160,6 +161,23 @@ test[0].foo = "b";
 test[0].bar.x = 2;
 ```
 
+### Buildable
+
+A combination of both `DeepWritable` and `DeepPartial`.
+
```

Comment:
```suggestion
This type allows building an object step by step by assigning the values to its attributes in multiple statements.
```

Reply:
Shall I update the example as well to something more similar to above?

Reply:
Indeed. It will look better. Maybe a comparison between two :)